floris/tools/__init__.py

```diff
@@ -46,18 +46,22 @@
     visualize_cut_plane,
     visualize_quiver,
 )
-from .wind_rose import WindRose
+from .wind_data import (
+    TimeSeries,
+    WindDataBase,
```

Comment:
I don't think users will be using this base-class, they'll probably only use the derived classes. So we can remove `WindDataBase` from the user-facing imports.

Reply:
Done!